projects/schematics/README.md

```diff
@@ -14,12 +14,21 @@ Navigate to `$ cd projects/schematics` and install the dependencies using `$ yar
 
 ### Unit testing
 
-To run schematics unit tests:
+To run all the schematics unit tests:
 
 1. `$ cd projects/schematics`
 2. `$ yarn` - to install dependencies
 3. `$ yarn test`
 
+If you wish to unit test a migration task, you can do it in the following way:
```

Comment:
This is something that took me some time to grasp, I don't know if it would be relevant to add (I thought we needed to commit unit tests for each migration scenario):

The schematics already have unit tests to cover the migration task they were designed to perform.  However, you might want to test if the new schematics configuration you added will produce the expected result when a user will perform a migration with the help of the schematics without running a full migration on an app, which would be very time consuming.  A convenient way to test you new config is to temporarily modify a schematics unit test case and use an example that will use your new config instead.  After you assess your migration scenario plays out as expected, you can revert the changes you did in the unit test.

Reply:
Done, I copied your suggestion 👍